### PR TITLE
Work bins patch 1

### PR DESCRIPTION
**Pull request by bwgabel**
Adding Work Bins to the Basecamp 3 extras page.

### CODE REVIEW

**Comment by bwgabel** [comment]
Is there anything else I need to do to be added to the Integrations page? I submitted this pull request 22 days ago and have not received any feedback. Am I missing something??

File: integrations.yml [hunk 1]
```diff
@@ -143,3 +143,9 @@
   website: "https://www.ziflow.com/basecamp-integration"
   icon: "ziflow.png"
   category: 6
+-
+  name: "Work Bins Outlook Plugin"
+  description: "Link email conversations to Basecamp projects and post email content to a discussion or a to-do."
+  website: "https://www.work-bins.com/"
```
Comment (javan) [inline integrations.yml]: I'm getting an SSL error visiting this URL. Did you intend to list it with `https`?

**Comment by bwgabel** [comment]
Yes, my software is compatible with Basecamp 3. I'll be sure to update the website to reflect that.

Thanks,
Brandon Gabel



Sent via the Samsung Galaxy S® 6, an AT&T 4G LTE smartphone


-------- Original message --------

**Comment by bwgabel** [comment]
It is suposed to be https. Apparently my host is experiencing a dns issue right now. I'll let you know once it's is resolved.



Sent via the Samsung Galaxy S® 6, an AT&T 4G LTE smartphone


-------- Original message --------

**Comment by bwgabel** [comment]
Javan,

The DNS issue is resolved and the website it working. Depending on your location it may take a little while for the dns changes to propagate so that you are directed to the new site.  

This was bad timing on my hosts part!

Thanks,
Brandon

**Pull request merged**